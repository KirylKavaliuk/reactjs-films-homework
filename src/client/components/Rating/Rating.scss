@import 'styles/palette';

.rating {
  display: flex;
  box-sizing: border-box;
  width: 250px;
  padding: 20px 0;
  justify-content: space-between;
}

.stars {
  display: flex;
  width: 200px;
  justify-content: space-around;
}

.star {
  width: 30px;
  height: 30px;
  fill: $white;
}

.active {
  fill: $light_blue;
}

.value-wrap {
  display: flex;
  justify-content: center;
  align-items: center;

  &:hover {
    cursor: default;
  }
}

.value {
  box-sizing: border-box;
  padding: 3px 6px;
  border: 1px solid #09ade3;
  border-radius: 2px;
  font-size: 16px;
}
