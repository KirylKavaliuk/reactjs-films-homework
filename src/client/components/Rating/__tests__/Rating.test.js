import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Rating from '../Rating';

it('renders rating shallow correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Rating value={ 3.3 }/>);

  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});

it('renders rating with value bigger than 5 shallow correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Rating value={ 1356.36 }/>);

  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
