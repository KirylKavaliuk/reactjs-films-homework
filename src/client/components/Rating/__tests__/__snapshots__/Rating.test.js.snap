// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rating/> 1`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 2`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 3`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 4`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 5`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 6`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 7`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 8`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 9`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 10`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 11`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 12`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 13`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 14`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 15`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4 Rating__active___BzsSn"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      2.50
    </div>
  </div>
</div>
`;

exports[`<Rating/> 16`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 17`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 18`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 19`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 20`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 21`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 22`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 23`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 24`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 25`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 26`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 27`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 28`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 29`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;

exports[`<Rating/> 30`] = `
<div
  className="Rating__rating___3f4Qx"
>
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__star___3Zoo4"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  />
  <div
    className="Rating__rating-number-wrap___dhLPE"
  >
    <div
      className="Rating__rating-number___13U9X"
    >
      0.00
    </div>
  </div>
</div>
`;
